shopelectro/settings/base.py

```diff
@@ -465,6 +465,7 @@ def get_robots_content():
     ],
 }
 
+# add/exclude logic works only for root items
```

Comment:
It seems that the `add` logic can work with any elements. Please let's cover it in tests